frontends/blog/Blog.tsx

```diff
@@ -1,37 +1,66 @@
-import { Blog, Button, Head, Modal, TabBody, TabsHeader } from 'components';
+import { Alert, Blog, Button, Head, Modal, TabBody, TabsHeader } from 'components';
 import { useDevice } from 'hooks';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 import { FC, useEffect, useState } from 'react';
-import { useFollowBlogMutation, useUnfollowBlogMutation } from 'store/apis';
+import {
+  useCheckSubscriptionQuery,
+  useSubscribeMutation,
+  useUnSubscribeMutation,
+} from 'store/apis';
 import { addAmazonUri, convertBlogToHeadProp, get } from 'utils';
-import { BLOG_TAB_MENUS, BLOG_TABS, ICONS } from 'variables';
+import { BLOG_TAB_MENUS, BLOG_TABS, ICONS, TELEGRAM_BOT } from 'variables';
 
 import styles from './Blog.module.scss';
 import { IBlogPageProps } from './Blog.types';
 
 const HeartIcon = ICONS.heart;
 
-export const BlogPage: FC<IBlogPageProps> = ({ blog, error, fullUrl }: IBlogPageProps) => {
-  const [isUnfollowModalOpen, setIsUnfollowModalOpen] = useState<boolean>(false);
-  const [isFollowed, setIsFollowed] = useState<boolean>(blog?.isFollowed || false);
-  const { isMobile } = useDevice();
-  const [followBlog, followBlogRes] = useFollowBlogMutation();
-  const [unfollowBlog, unfollowBlogRes] = useUnfollowBlogMutation();
+export const BlogPage: FC<IBlogPageProps> = ({ blog, error, fullUrl }) => {
   const {
     query: { id },
   } = useRouter();
+  const [isUnfollowModalOpen, setIsUnfollowModalOpen] = useState<boolean>(false);
+  const { isMobile } = useDevice();
+  const [subscribeBlog, subscribeBlogRes] = useSubscribeMutation();
+  const [unSubscribeBlog, unSubscribeBlogRes] = useUnSubscribeMutation();
+  const { data: isSubscribed, isError: checkIsSubscriptionErr } = useCheckSubscriptionQuery(id);
+  const [isFollowed, setIsFollowed] = useState<boolean>(isSubscribed);
```

Comment:
change follow words to subscribe
ex: isFollowed -> isSubscribed

Reply:
actually we do not need this state, we can use response from check-subscription/${id} api